modules/table/src/main/java/org/apache/ignite/internal/table/distributed/StorageUpdateHandler.java

```diff
@@ -417,4 +417,12 @@ public void buildIndex(UUID indexId, List<UUID> rowUuids, boolean finish) {
     public PendingComparableValuesTracker<HybridTimestamp, Void> getSafeTimeTracker() {
         return safeTimeTracker;
     }
+
+    /**
+     * Waits for indexes to be created.
+     */
+    // TODO: IGNITE-19513 Fix it, we should have already waited for the indexes to be created
```

Comment:
Again, the referred ticket description is basically nonexistent

Reply:
Try to fix it

Reply:
it's better now